setup-protoc/src/installer.ts

```diff
@@ -208,3 +212,15 @@ function normalizeVersion(version: string): string {
 
   return version;
 }
+
+function includePrerelease(
+  isPrerelease: boolean,
+  includePrereleases: boolean
+): boolean {
+  if (!includePrereleases) {
+    if (isPrerelease) {
+      return false;
+    }
+  }
+  return true;
```

Comment:
this could be `return includePrereleases || !isPrerelease`

Reply:
I thought a function could be more explicit, but we can replace it with boolean logic + comment :+1:

Reply:
I usually prefer explicit code as well but in this case the one-liner is pretty readable IMO